src/Mod/TemplatePyMod/FeaturePython.py

```diff
@@ -155,7 +156,8 @@ def __init__(self, obj):
 		obj.addProperty("App::PropertyLength","Height","Octahedron", "Height of the octahedron").Height=1.0
 		obj.addProperty("Part::PropertyPartShape","Shape","Octahedron", "Shape of the octahedron")
 		obj.Proxy = self
- 
+		obj.recompute()

```

Comment:
It is a bad idea to recompute the objects inside the class, the recomputation should happen after the object is created.

I don't remember a specific thread where this was discussed but the idea is that if you add recomputations within the class, it may slow down the operation of the software if you have to do many recalculations.

It should be done like this.
```py
obj = make_something()  # New scripted object is generated
App.activeDocument().recompute()
```

Reply:
Okay.